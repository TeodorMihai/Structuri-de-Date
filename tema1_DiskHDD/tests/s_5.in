2 1
::r 0 15
2 
::w 0 15 0015
2
::d 0 15
1
::r 0 14
2 
::w 0 14 0014
2
::r 0 13
2
::w 0 13 0013
2
::r 0 12
2 
::w 0 12 0012
2
::r 0 12
2 
::w 0 12 0012
2
::r 0 11
2 
::w 0 11 0011
2
::r 0 10
2
::w 0 10 0010
2
::d 0 10
1
::w 0 08 0008
200
::e
