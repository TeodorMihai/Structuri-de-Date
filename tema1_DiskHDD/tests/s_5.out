0000
40
0011
0012
0012
0013
0014
3
0015
2.75 3.00 30.00 91.50
